test/core/fixtures/transformation/es6.template-literals/only/expected.js

```diff
@@ -1,3 +1,3 @@
 "use strict";
 
-var foo = "" + test;
+var foo = test;
```

Comment:
Ok I can make it back to `"" + test` or do `String(test)` but then `es6.spec.template-literals` will still be `"" + String(test)` unless I do some check.